### ISSUE TITLE
Uncaught TypeError: Cannot read property 'prototype' of undefined

### ISSUE THREAD

**Issue by ajclarke90** [atom/tree-view#314]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.166.0
**System**: Mac OS X 10.10.1
**Thrown From**: [tree-view](https://github.com/atom/tree-view) package, v0.145.0

### Stack Trace

Uncaught TypeError: Cannot read property 'prototype' of undefined

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:5

TypeError: Cannot read property 'prototype' of undefined
  at __extends (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:5:199)
  at /Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:41:5
  at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:1087:5)
  at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:1089:4)
  at Module._compile (module.js:468:26)
  at Object.Module._extensions..js (module.js:486:10)
  at Module.load (/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-script/lib/coffee-script/register.js:45:36)
  at Function.Module._load (module.js:318:12)
  at Module.require (module.js:373:17)
  at require (module.js:392:17)
  at Object.module.exports.createView (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/main.js:108:20)
  at atom-workspace.disposables.add.atom.commands.add.tree-view:toggle (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/main.js:45:26)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:549:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:391:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:167:20)

```

### Commands

```
     -3:42.5 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused)
     -3:28.2 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.remove)
     -0:00.0 tree-view:toggle (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ]
  }
}
```

### Installed Packages

```coffee
# User
build, v0.22.1

# Dev
No dev packages
```

/cc @atom/core

**Comment by jorgeavaldez** [comment]
I'm having the same issue on Windows 8.1.

I can open atom from PowerShell in safe modeby running
`atom --safe`

Here's my output.

`TypeError: Cannot read property 'prototype' of undefined
  at __extends (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\node_modules\image-view\lib\image-editor-status-view.js:4:199)
  at c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\node_modules\image-view\lib\image-editor-status-view.js:13:5
  at Object.<anonymous> (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\node_modules\image-view\lib\image-editor-status-view.js:84:5)
  at Object.<anonymous> (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\node_modules\image-view\lib\image-editor-status-view.js:86:4)
  at Module._compile (module.js:468:26)
  at Object.Module._extensions..js (module.js:486:10)
  at Module.load (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\node_modules\coffee-script\lib\coffee-script\register.js:45:36)
  at Function.Module._load (module.js:318:12)
  at Module.require (module.js:373:17)
  at require (module.js:392:17)
  at createImageStatusView (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\node_modules\image-view\lib\main.js:28:31)
  at c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\node_modules\image-view\lib\main.js:16:16
  at Emitter.module.exports.Emitter.emit (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\node_modules\event-kit\lib\emitter.js:82:11)
  at PackageManager.module.exports.PackageManager.activate (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\src\package-manager.js:383:27)
  at Atom.module.exports.Atom.startEditorWindow (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\src\atom.js:605:21)
  at Object.<anonymous> (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\src\window-bootstrap.js:12:8)
  at Object.<anonymous> (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\src\window-bootstrap.js:23:4)
  at Module._compile (module.js:468:26)
  at Object.Module._extensions..js (module.js:486:10)
  at Module.load (c:\Users\Jorge\AppData\Local\atom\app-0.166.0\resources\app\node_modules\coffee-script\lib\coffee-script\register.js:45:36)
  at Function.Module._load (module.js:318:12)
  at Module.require (module.js:373:17)
  at require (module.js:392:17)
  at window.onload (file:///C:/Users/Jorge/AppData/Local/atom/app-0.166.0/resources/app/static/index.js:59:25)
 `

I uninstalled, deleted all of the remaining folders and files, and then reinstalled using the installer, which did not help.

**Comment by kevinsawicki** [comment]
@ajclarke90 and @jorgeavaldez can you upgrade to Atom 0.172 and see if you still see this issue?

**Comment by ajclarke90** [comment]
@kevinsawicki - Sorry, since fixing an other issue (referenced above) by uninstalling and reinstalling the build package, I haven't been able to reproduce this error.

**Comment by izuzak** [comment]
@jorgeavaldez can you still reproduce this issue in Atom 0.178.0 in safe mode?

**Comment by kevinsawicki** [comment]
I believe this has been fixed by subsequent space pen versions, the 1.0 API, and use of asar for bundling all the core files, closing this out.
